.github/workflows/ci.yml

```diff
@@ -26,26 +26,18 @@ jobs:
     name: test
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/cache@v3
-        with:
-          path: |
-            deps
-            _build
-          key: ${{ runner.os }}-mix-${{ matrix.elixir_version }}-${{ hashFiles('**/mix.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-mix-
```

Comment:
I chose to remove this cache because it's much faster to rebuild the deps then restoring the cache. The only exception is the build of the native code.